### ISSUE TITLE
Accessibility: Change UNSAVED file icon used on tab-bar / Doc Switcher for color-impaired users

### ISSUE THREAD

**Issue by sasumner** [notepad-plus-plus/notepad-plus-plus#8071]
A color-blind person has reported extreme difficulty in telling apart the "saved file" and "unsaved file" icons on the tab bar: https://community.notepad-plus-plus.org/topic/19110/different-indicator-for-unsaved-tabs

This should be resolved; it shows respect for those with accessibility concerns.

Issue #8068 raises this complaint for BOTH icons, but it is being split into multiple issues so that individual PRs can be accepted or rejected on a case-by-case basis.

See #8068 to find a more complete description and the corresponding companion issues.

This is being reported against Notepad v. 7.8.5 but this icon has been in its current state for a long time.

**Comment by donho** [comment]
Please make it optional. ie. keep the current icons as default and color blind people can switch to the icon set they need.

**Comment by sasumner** [comment]
Sorry for the schizophrenia with the daughter/parent issues; the thinking was it might be easier to judge each proposed icon on its own merits individually; thus the reason for the 3 daughter issues to the main issue.

Now that @donho has weighed in and requested a "color blind" "icon set" preference (not that we'll call it that), I think this is better as a SINGLE issue, so I'm closing this (and the other "daughters") and reopening the parent.
